src/accelerate/accelerator.py

```diff
@@ -156,8 +159,9 @@ def __init__(
         rng_types: Optional[List[Union[str, RNGType]]] = None,
         log_with: Optional[List[Union[str, LoggerType, GeneralTracker]]] = None,
         logging_dir: Optional[Union[str, os.PathLike]] = None,
+        downcast_bf16: Optional[bool] = False,
         dispatch_batches: Optional[bool] = None,
-        step_scheduler_with_optimizer: bool = True,
+        step_scheduler_with_optimizer: Optional[bool] = True,
```

Comment:
Doesn't seem to be able to be None, so I'm confused about this change in typing.